src/components/ErrorBoundary.jsx

```diff
@@ -3,14 +3,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import postErrorToSlack from 'utils/postErrorToSlack';
 import { connect } from 'react-redux';
-import Error from 'pages/_error';
 
 // Implementation of https://reactjs.org/docs/error-boundaries.html
 // Using React class because it's not yet supported for functional components
 // according to https://stackoverflow.com/a/68075800/1940886
 class ErrorBoundary extends React.Component {
   constructor(props) {
     super(props);
+    // eslint-disable-next-line react/no-unused-state
```

Comment:
why disabling the linter instead of removing the state? afaict `this.state` is no longer used

Reply:
If I remove this linter disable, the linter will complain that the constructor is useless and that it should be removed. We still need to call the parent's constructor to be able to set up the required functionalities.

I've removed this linter and the state, but need to add another linter command to stop the linter complaining of empty constructor.